### PR TITLE
[FR] Update `Ranking_Criteria\osu!mania`

### PR DESCRIPTION
**Pull request by DoryanR**
Proposal to update the French translation of the wiki page `Ranking_Criteria\osu!mania`.

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 1]
```diff
@@ -1,93 +1,204 @@
----
-outdated: true
----
+# Critères de classement d'osu!mania
 
-# Critères de Ranking osu!mania
+Les **critères de classement d'osu!mania** sont des règles et des directives qui s'appliquent à la création des difficultés spécifiques à osu!mania. Pour qu'une difficulté spécifique à osu!mania soit classée, il est obligatoire que la création obéisse aux critères énumérés. Bien que **toutes les règles doivent être suivies en toute circonstance**, les directives peuvent être ignorées dans des circonstances exceptionnelles. Ces circonstances exceptionnelles doivent être justifiées par une explication exhaustive de la raison pour laquelle la directive a été ignorée et pourquoi le fait de ne pas l'ignorer nuira à la qualité globale de la création.
+
+Les propositions de règles ainsi que les suggestions sont discutées dans le [Sous-forum Ranking Criteria] (https://osu.ppy.sh/community/forums/87). Toute règle ayant fait l'objet d'une discussion et d'un processus d'approbation par la communauté est listée ici car elle a été approuvée dans le fil de discussion correspondant.
```
Comment (DoryanR) [inline wiki/Ranking_Criteria/osu!mania/fr.md:5]:
```suggestion
Les propositions de règles ainsi que les suggestions sont discutées dans le [Sous-forum Ranking Criteria](https://osu.ppy.sh/community/forums/87). Toute règle ayant fait l'objet d'une discussion et d'un processus d'approbation par la communauté est listée ici car elle a été approuvée dans le fil de discussion correspondant.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 2]
```diff
@@ -1,93 +1,204 @@
...
+### ![](/wiki/shared/diff/hard-m.png?20211215) Hard
 
-Voici un exemple comportant des jackhammers : [MENDES Black Another](https://www.youtube.com/watch?v=qqkWIWbO14c).
+#### Directives
 
-- **Pour les difficultés non-expert, vous pouvez utiliser des jackhammers *seulement si* le BPM est inférieur à 150 et *si* ils correspondent à la musique.** Par exemple, dans des musiques de dubstep ou autre genre à rythme frénétique.
-- **Pour les difficultés expert, vous pouvez utiliser les jackhammers sans limite de BPM tant que les jackhammers respectent le rythme.**
+- **Les valeurs HP et OD ne doivent pas être supérieures à 8.**
+- **Il convient de ne pas utiliser de snapping de note de 1/8 consécutif et plus.** Ceux-ci peuvent être trop exigeants physiquement pour les joueurs de ce niveau. Une exception à cette règle serait l'utilisation de 3 grace qui sont à des niveaux de snaps  plus élevés (1/8, 1/12 et 1/16) car elles ne sont utilisées que pour accentuer les sons des graces.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:151]:
```suggestion
- **Il convient de ne pas utiliser de snapping de note de 1/8 consécutif et plus.** Ceux-ci peuvent être trop exigeants physiquement pour les joueurs de ce niveau. Une exception à cette règle serait l'utilisation de 3 graces qui sont à des niveaux de snaps plus élevés (1/8, 1/12 et 1/16) car elles ne sont utilisées que pour accentuer les sons des graces.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 3]
```diff
@@ -1,93 +1,204 @@
...
+- **Les séries de long streams de 1/4 doivent inclure, au minimum, une demi-pause lorsqu'il est naturel de le faire après chaque phrase musicale.** Ceux-ci peuvent être trop exigeants physiquement pour les joueurs de ce niveau.
+- **Les chords à l'intérieur d'un quart de stream doivent être espacés d'au moins 1/1 temps les uns des autres.** Une utilisation plus dense des chords est trop difficile pour les joueurs de ce niveau de compétence.
+- **Évitez d'utiliser des chords au milieu des streams1/6.** Toutefois, l'utilisation d'un chord au début ou à la fin d'un stream est acceptable.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:154]:
```suggestion
- **Évitez d'utiliser des chords au milieu des streams 1/6.** Toutefois, l'utilisation d'un chord au début ou à la fin d'un stream est acceptable.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 4]
```diff
@@ -1,93 +1,204 @@
...
+- **L'utilisation de 1/4 d'un jack est déconseillée.** Ceux-ci peuvent être trop exigeants physiquement pour les joueurs de ce niveau. Des minijacks entrecoupés d'un temps de repos suffisant peuvent être acceptés.
+- **Évitez d'utiliser plus de 9 notes consécutives dans un trill.** La coordination et l'équilibre des mains requis pour ce pattern sont trop difficiles pour les joueurs de ce niveau.
+- **Les long notes doivent être hold pendant au moins 1/4 de temps.**
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:157]:
```suggestion
- **Les long notes doivent être tenues pendant au moins 1/4 de temps.**
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 5]
```diff
@@ -1,93 +1,204 @@
...
-- **L'Overall Difficulty devrait être réglé sur :**
-  - 9 ou 10 pour une map composée presque exclusivement de Hits.
-  - 7 ou 8 pour une map composée de Holds et de Hits de façon équilibrée.
-  - 5 ou 6 pour une map composée de nombreux Holds, telles que les maps Insane des jeux O2.
-- **N'utilisez que les *breaks* lorsque c'est nécéssaire** (long silence, pas l'instrument que vous mappez, etc...)
-- **Il est conseillé d'avoir au moins deux nombres de colonnes dans votre mapset**, mais il est tout à fait acceptable de ne faire que du 4K ou du 6K si vous le souhaitez.
-- **Les *SV changes* devraient être utilisés avec parcimonie.** Utilisez-les lorsqu'il y a un changement notable dans la musique. Tout dépend du mapper.
+Directives supplémentaires pour les difficultés *7 key Hard* :
+
+- **Évitez les chords de plus de 4 notes.** Cela inclut les longs pattern de notes qui impliquent des notes sur plus de quatre colonnes.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:169]:
```suggestion
- **Évitez les chords de plus de 4 notes.** Cela inclut les longs patterns de notes qui impliquent des notes sur plus de quatre colonnes.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 6]
```diff
@@ -1,93 +1,204 @@
...
+- **Les objets ne doivent pas être placés pendant la durée de hold d'une long note 1/2.** Cependant, l'utilisation d'un objet au début ou à la fin d'une long note est acceptable.
+- **Évitez les rythmes plus rapides que 1/2 pendant la durée de hold d'une long note de 1/1.**
+- **Évitez d'utiliser toute forme de pattern shield 1/4.** La coordination requise pour ce pattern est trop difficile pour les joueurs de ce niveau.
+- **La densité des notes doit consister principalement en 1/1 et 1/2, occasionnellement 1/4, ou des rythmes plus lents.**
+- **Évitez d'utiliser plus de 5 notes 1/4 consécutives.**
+- **Les longs streams de 1/2 notes doivent inclure, au minimum, une pause de 1/1 lorsqu'il est naturel de le faire après chaque phrase musicale.**, car elles peuvent devenir trop exigeantes physiquement pour les joueurs de ce niveau.
+- **Évitez d'utiliser des chords pendant des pattern consécutifs de 1/4 de note.** Cependant, l'utilisation d'un chord pour commencer ou terminer un pattern est acceptable.
+- **Évitez les anchors composés de trois notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longues sont éprouvantes.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:131]:
```suggestion
- **Évitez les anchors composés de trois notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longs sont éprouvants.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 7]
```diff
@@ -1,93 +1,204 @@
...
+- **Les changements de long-term slider velocity devraient se situer entre 0,8x et 1,10x.**
+- **Les slider velocity gimmicks devraient pouvoir réagir dans 1/2 temps ou plus.**
 
-## Conseils
+Directives supplémentaires pour les difficultés *4 key Hard* :
 
-Ces informations sont importantes et devraient être suivis par la plupart des mappers. Cependant, ce ne sont pas des règles, ce qui veut dire qu'un mapper peut choisir de les ignorer sous certaines circonstances. Si vous êtes sur le point d'ignorer un conseil, posez vous des questions : "Est-ce que ce que je m'apprête à apporter quelque chose de positif à ma map ?" Soyez certain de ce que vous faites.
+- **Évitez les chords de plus de 3 notes.** Cela inclut les longs pattern de notes qui impliquent des notes sur plus de trois colonnes.
+- **Évitez les anchors composés de cinq notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longues sont éprouvantes.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:165]:
```suggestion
- **Évitez les anchors composés de cinq notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longs sont éprouvants.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 8]
```diff
@@ -1,93 +1,204 @@
...
-  - 9 ou 10 pour une map composée presque exclusivement de Hits.
-  - 7 ou 8 pour une map composée de Holds et de Hits de façon équilibrée.
-  - 5 ou 6 pour une map composée de nombreux Holds, telles que les maps Insane des jeux O2.
-- **N'utilisez que les *breaks* lorsque c'est nécéssaire** (long silence, pas l'instrument que vous mappez, etc...)
-- **Il est conseillé d'avoir au moins deux nombres de colonnes dans votre mapset**, mais il est tout à fait acceptable de ne faire que du 4K ou du 6K si vous le souhaitez.
-- **Les *SV changes* devraient être utilisés avec parcimonie.** Utilisez-les lorsqu'il y a un changement notable dans la musique. Tout dépend du mapper.
+Directives supplémentaires pour les difficultés *7 key Hard* :
+
+- **Évitez les chords de plus de 4 notes.** Cela inclut les longs pattern de notes qui impliquent des notes sur plus de quatre colonnes.
+- **Évitez les anchors composés de cinq notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longues sont éprouvantes.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:170]:
```suggestion
- **Évitez les anchors composés de cinq notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longs sont éprouvants.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 9]
```diff
@@ -1,93 +1,204 @@
...
+- **Évitez d'utiliser des split rolls ou toute sorte de stream 1/8 compliqués qui durent plus de 4 temps.** Les streams de ce type sont difficiles à exécuter et ne doivent être utilisés que si la musique le justifie.
+- **Évitez d'utiliser 1/6 ou plus pour les streams de long notes.** Ceux-ci sont très difficiles à toucher et à relâcher en raison de leur très courte longueur. Utilisez plutôt des notes régulières à la place des streams.
+- **Les changements de long-term slider velocity devraient se situer entre 0,70x et 1,10x.**
+- **Les slider velocity gimmicks doivent pouvoir réagir à 1/4 de temps ou plus lorsqu'ils ne sont pas normalisés et à 1/2 de temps ou plus lorsqu'ils sont normalisés.**
+
+Directives supplémentaires pour les difficultés *4 key Insane* :
+
+- **Évitez d'utiliser des chords au milieu des 1/8.** La densité de ces pattern est souvent trop difficile pour les joueurs de ce niveau de compétence, surtout lorsqu'ils sont utilisés sur la difficulté la plus faible d'une beatmap. De plus, un placement inefficace des chords peut entraîner une utilisation incohérente des minijacks 1/4. L'utilisation d'un chord au début ou à la fin d'un stream est acceptable.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:186]:
```suggestion
- **Évitez d'utiliser des chords au milieu des 1/8.** La densité de ces patterns est souvent trop difficile pour les joueurs de ce niveau, surtout lorsqu'ils sont utilisés sur la difficulté la plus faible d'une beatmap. De plus, un placement inefficace des chords peut entraîner une utilisation incohérente des minijacks 1/4. L'utilisation d'un chord au début ou à la fin d'un stream est acceptable.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 10]
```diff
@@ -1,93 +1,204 @@
...
-- **N'utilisez que les *breaks* lorsque c'est nécéssaire** (long silence, pas l'instrument que vous mappez, etc...)
-- **Il est conseillé d'avoir au moins deux nombres de colonnes dans votre mapset**, mais il est tout à fait acceptable de ne faire que du 4K ou du 6K si vous le souhaitez.
-- **Les *SV changes* devraient être utilisés avec parcimonie.** Utilisez-les lorsqu'il y a un changement notable dans la musique. Tout dépend du mapper.
+Directives supplémentaires pour les difficultés *7 key Hard* :
+
+- **Évitez les chords de plus de 4 notes.** Cela inclut les longs pattern de notes qui impliquent des notes sur plus de quatre colonnes.
+- **Évitez les anchors composés de cinq notes ou plus.** Bien qu'elles soient faciles à lire pour les joueurs, les anchors plus longues sont éprouvantes.
+- **Maintenez la distance entre les chords dans un chordstream à un intervalle de 1/1 ou plus.** Tout ce qui est plus dense que cela peut nuire à la lisibilité.
+- **Les hands occasionnelles peuvent être utilisées dans les streams, à condition qu'elles soient séparées par au moins une mesure (4 temps).**
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:172]:
```suggestion
- **Les hands peuvent être utilisés occasionnellement dans les streams, à condition qu'ils soient séparés par au moins une mesure (4 temps).**
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 11]
```diff
@@ -1,93 +1,204 @@
...
+- **Les changements de long-term slider velocity devraient se situer entre 0,70x et 1,10x.**
+- **Les slider velocity gimmicks doivent pouvoir réagir à 1/4 de temps ou plus lorsqu'ils ne sont pas normalisés et à 1/2 de temps ou plus lorsqu'ils sont normalisés.**
+
+Directives supplémentaires pour les difficultés *4 key Insane* :
+
+- **Évitez d'utiliser des chords au milieu des 1/8.** La densité de ces pattern est souvent trop difficile pour les joueurs de ce niveau de compétence, surtout lorsqu'ils sont utilisés sur la difficulté la plus faible d'une beatmap. De plus, un placement inefficace des chords peut entraîner une utilisation incohérente des minijacks 1/4. L'utilisation d'un chord au début ou à la fin d'un stream est acceptable.
+- **Il n'est pas recommandé d'utiliser les split-jumptrills pour les temps de 2 ou plus qui mènent à un hand.** À moins qu'une beatmap n'utilise fréquemment des jacks ou d'autres patterns complexes, ce pattern est difficile à frapper. Si un hand est nécessaire, supprimez la note qui la précède et qui créerait le mini jack.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:187]:
```suggestion
- **Il n'est pas recommandé d'utiliser les split-jumptrills pour les temps de 2 ou plus qui mènent à un hand.** À moins qu'une beatmap n'utilise fréquemment des jacks ou d'autres patterns complexes, ce pattern est difficile à jouer. Si un hand est nécessaire, supprimez la note qui la précède et qui créerait le mini jack.
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 12]
```diff
@@ -1,93 +1,204 @@
...
+- **Évitez d'utiliser plus de 9 notes pour les split-jumptrills.** Ceux-ci sont plus épuisants physiquement que d'autres patterns de densité similaire.
+
+Directives supplémentaires pour les difficultés *7 key Insane* :
+
+- **Évitez d'utiliser les jacks 1/4 avec 3 notes ou plus au milieu d'un chordstream.** Ceux-ci exigent une souplesse des doigts qui est trop difficile pour les joueurs de ce niveau.
+- **Au milieu d'un long chord stream avec un chord chaque...**
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:193]:
```suggestion
- **Au milieu d'un long chord stream avec un chord à chaque...**
```

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 13]
```diff
@@ -1,93 +1,204 @@
...
+- **Les objets situés directement après les changements de short-term slider velocity devraient apparaître à la vitesse de défilement normale de la beatmap.** Pour ce faire, les changements de short-term slider velocity ne doivent être utilisés que dans les espaces vides entre les notes et doivent s'égaliser à une distance de 1,0x.
+- **Les changements de long-term slider velocity devraient se situer entre 0,8x et 1,10x.**
+- **Les slider velocity gimmicks devraient pouvoir réagir dans 1/2 temps ou plus.**
 
-## Conseils
+Directives supplémentaires pour les difficultés *4 key Hard* :
 
-Ces informations sont importantes et devraient être suivis par la plupart des mappers. Cependant, ce ne sont pas des règles, ce qui veut dire qu'un mapper peut choisir de les ignorer sous certaines circonstances. Si vous êtes sur le point d'ignorer un conseil, posez vous des questions : "Est-ce que ce que je m'apprête à apporter quelque chose de positif à ma map ?" Soyez certain de ce que vous faites.
+- **Évitez les chords de plus de 3 notes.** Cela inclut les longs pattern de notes qui impliquent des notes sur plus de trois colonnes.
```
Comment (Ganondorf-1) [inline wiki/Ranking_Criteria/osu!mania/fr.md:164]:
```suggestion
- **Évitez les chords de plus de 3 notes.** Cela inclut les longs patterns de notes qui impliquent des notes sur plus de trois colonnes.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!mania/fr.md [hunk 14]
```diff
@@ -1,93 +1,204 @@
...
+  - **...entre 3:15 et 4:00**, la difficulté la plus faible de chaque key mode inclus ne peut pas être plus difficile qu'une Insane, **OU** chaque key mode doit fournir une répartition commençant au moins 1 niveau de difficulté en dessous de la difficulté la plus élevée.
+- Sur les difficultés Insane et plus difficiles, une bonne répartition est définie par des écarts de difficulté relative similaires à ceux définis formellement entre les difficultés inférieures.
 
-- **Il est recommendé de désactiver l'option *letterbox* dans l'onglet Design de la map.**
+### Directives
 
-### Echelle de difficulté
+- **Chaque note doit correspondre à un son présent dans la musique.** Il doit s'agir d'un son distinct, mais il peut aussi représenter de manière cohérente un son continu avec un début ou une fin indiscernable. Des exceptions en faveur de la simplification doivent être faites lorsque suivre la musique de manière tout à fait précise...
```
Comment (Biokax) [inline wiki/Ranking_Criteria/osu!mania/fr.md:71]:
```suggestion
- **Chaque note doit correspondre à un son présent dans la musique.** Il doit s'agir d'un son distinct, mais il peut aussi représenter de manière cohérente un son continu avec un début ou une fin indiscernable. Des exceptions à des fins de simplification doivent être faites lorsqu'il suit la musique de manière tout à fait précise...
```

**Review by Biokax — CHANGES_REQUESTED** [review]

File: wiki/Ranking_Criteria/osu!mania/fr.md (hunk 14 above)
Comment (Biokax) [reply wiki/Ranking_Criteria/osu!mania/fr.md:71]: je ne suis pas totalement sûr pour celui là, mais ça semble déjà plus compréhensible

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**